growth/too/templates/plan_new.html

```diff
@@ -71,7 +71,13 @@
                         <small id="balanceHelpBlock" class="form-text text-muted">
                         Require field observations in all requested filters
                         </small>
-
+                    </div>
+                    <div class="custom-control custom-switch" aria-describedby="balanceHelpBlock">
+                        {{ render_field(form.planned, class='custom-control-input') }}
+                        <label for=planned class=custom-control-label>Use tonight's plan</label>
```

Comment:
minor suggestion: 'Account for tonight's plan'
Also, would it be possible for you to add a short description to the use previous flag (mainly to help differentiate from doPlanned flag)? Observation planning is starting to get a bit complicated now on the user side :)